utils/network_utils.js

```diff
@@ -25,104 +25,104 @@ let MIN_PORT = 49152;
 let MAX_PORT = 65535;
 
 let getRandomPort = function() {
-	return Math.round(Math.random() * (MAX_PORT - MIN_PORT) + MIN_PORT);
+  return Math.round(Math.random() * (MAX_PORT - MIN_PORT) + MIN_PORT);
 }
 
 let NetworkUtils = {
-	/**
-	 * FIXME: should this just return ROS_IP?
-	 * get this computer's (non-internal) ip address
-	 * @param [family] {string} 'IPv4', 'IPv6', ... 'IPv4' default
-	 * @param [networkInterface] {string} network interface to use ('eth0') else finds first match
-	 */
-	getIpAddress: function(family, networkInterface) {
-		family = family || 'IPv4';
-		let interfaces = os.networkInterfaces();
-		let interfaceNames;
-		if (networkInterface && !ifaces.hasOwnProperty(networkInterface)) {
-			return null;
-		}
-		else if (networkInterface) {
-			interfaceNames = [ networkInterface ];
-		}
-		else {
-			interfaceNames = Object.keys(interfaces);
-		}
-
-		let ipAddress = null;
-		interfaceNames.some((ifName) => {
-		  interfaces[ifName].forEach((iface) => {
-		    if (iface.internal || family !== iface.family) {
-		      // skip over internal (i.e. 127.0.0.1) and addresses from different families
-		      return false;
-		    }
-
-				ipAddress = iface.address;
-				return true;
-		  });
-		});
-		return ipAddress;
-	},
-
-	/**
+  /**
+   * FIXME: should this just return ROS_IP?
+   * get this computer's (non-internal) ip address
+   * @param [family] {string} 'IPv4', 'IPv6', ... 'IPv4' default
+   * @param [networkInterface] {string} network interface to use ('eth0') else finds first match
+   */
+  getIpAddress: function(family, networkInterface) {
+    family = family || 'IPv4';
+    let interfaces = os.networkInterfaces();
+    let interfaceNames;
+    if (networkInterface && !ifaces.hasOwnProperty(networkInterface)) {
+      return null;
+    }
+    else if (networkInterface) {
+      interfaceNames = [ networkInterface ];
+    }
+    else {
+      interfaceNames = Object.keys(interfaces);
+    }
+
+    let ipAddress = null;
+    interfaceNames.some((ifName) => {
+      interfaces[ifName].forEach((iface) => {
+        if (iface.internal || family !== iface.family) {
+          // skip over internal (i.e. 127.0.0.1) and addresses from different families
+          return false;
+        }
+
+        ipAddress = iface.address;
+        return true;
+      });
+    });
+    return ipAddress;
+  },
+
+  /**
    * FIXME: should this just return ROS_HOSTNAME
```

Comment:
I don't think so currently - whether to use this or not is an option you can set when you initialize the node.